src/main/java/org/team1540/robot2023/commands/drivetrain/AutoSideBalanceCommand.java

```diff
@@ -0,0 +1,48 @@
+package org.team1540.robot2023.commands.drivetrain;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.team1540.robot2023.utils.AverageFilter;
+
+import java.util.function.Supplier;
+
+public class AutoSideBalanceCommand extends CommandBase {
```

Comment:
can probably combine this into the normal AutoBalanceCommand and take whether or not to side balance as an argument